docs/tutorials/basic/data.md

```diff
@@ -125,37 +49,41 @@ for batch in data_iter:
     print([batch.data, batch.label, batch.pad])
 ```
 
-## Read CSV
-
-There is an iterator called `CSVIter` to read data batches from CSV files. We
-first dump `data` into a csv file, and then load the data.
+## Reading data from CSV files
+MXNet provides [`CSVIter`](http://mxnet.io/api/python/io.html#mxnet.io.CSVIter)
+to read from CSV files and can be used as below:
 
 ```python
+#lets save `data` into a csv file first and try reading it back
 np.savetxt('data.csv', data, delimiter=',')
 data_iter = mx.io.CSVIter(data_csv='data.csv', data_shape=(3,), batch_size=30)
 for batch in data_iter:
     print([batch.data, batch.pad])
 ```
 
```

Comment:
I am having a dataset and I want to pass it to mxnet, I tried to put it in a csv formate(considered the flattening issue) and call it by CSViter, but I should pass the csv label too, and when I do that the kernel(at anaconda) crash always!
What should I do!

can you add the next step after creating the csv iterator, which is passing that iterator to mxnet as the training or testing set, please? Thanks